packages/ui/src/machines/actors/auth/signIn.ts

```diff
@@ -390,7 +390,10 @@ export const signInActor = createMachine<SignInContext, AuthEvent>(
         const source = !!(context.intent && context.intent === 'autoSignIn')
           ? context.authAttributes
           : context.formValues;
-        const { username, password } = source;
+        let { country_code, username, password } = source;
+
+        username = (country_code ?? '') + username;
```

Comment:
nit - would prefer to make the destructured variables as `const` and just make a new value for the `username` on line 395. Won't block over it, though.